app/controllers/users_controller.rb

```diff
@@ -0,0 +1,3 @@
+class UsersController < ApplicationController
+  def index; end
+end
```

Comment:
- [ ] kindly add a show action to your controller
`def show; end`

Reply:
Thank you @VICTOR-CHUKWU for this suggestion :+1: